include/swift/AST/DiagnosticsSema.def

```diff
@@ -4980,6 +4980,12 @@ ERROR(capture_across_type_decl,none,
 ERROR(reference_to_invalid_decl,none,
       "cannot reference invalid declaration %0", (const ValueDecl *))
 
+WARNING(implicit_nonstrong_to_strong_capture,none,
+      "%0 capture list item %base1 implicitly captured as strong reference in outer scope",
```

Comment:
question regarding the output formatting of this: the reference ownership value appears to get quoted, which IMO makes the output a bit odd. is there some way to suppress that or customize it without too much duplication of whatever is currently responsible for producing the display string?

Reply:
IMO the quoting makes sense since it's directly referring to a keyword written in the code